Makefile

```diff
@@ -89,9 +89,10 @@ endif
 
 lint-composer:
 ifndef COMPOSER
-$(error "composer is not installed: https://getcomposer.org/doc/00-intro.md#installation-linux-unix-macos")
```

Comment:
Unrelated to this PR. Under normal circumstances I'd suggest removing these changes and adding to separate PR...